uber/decorators.py

```diff
@@ -248,7 +248,9 @@ def returns_json(*args, **kwargs):
 
 
 def multifile_zipfile(func):
-    func.site_mappable = True
+    parameters = inspect.getargspec(func)
```

Comment:
This is copied from our XLSX and CSV decorators -- it marks a ZIP export for the sitemap viewing only if it has the three standard parameters (self, out, session).